tests/scripts/custodian/README.md

```diff
@@ -0,0 +1,16 @@
+# QA Cloud Custodian
+
+QA custodian will clean up resources in AWS, GCR, and AZURE. Currently, we have:
+* AWS
+  * instances
+  * NLBs
+
+### Getting Started
+New Hires - make a PR adding one line to the `user-keys.txt` file. Use that key in ALL resources you create when using SUSE resources.  Keep it short (6 characters or less) unique to you, and recognizable by others (And, this is case sensitive!). For example, if your name was Melissa Di Donato, something like `mdd` would suffice. Just make sure you can remember it, and others can know who's it is if they are on your team. 
```

Comment:
Maybe update the example name to like John Doe or something? For....reasons 😄

Reply:
good idea, done :)